.github/workflows/pr_autoupdate.yaml

```diff
@@ -56,10 +51,8 @@ jobs:
         with:
           extra_args: --hook-stage manual --all-files
 
-      - name: Commit
```

Comment:
@Lancetnik could you please explain, how it works? Or show example PRs with autofixes

I dont't understand, if pre-commit makes any changes, where these changes go? Where this commit will appear, directly in fork?

I am not so experienced in Github Actions, so I try to build strong understanding of current workflow to make sure anything will work as it was before